src/transformations.py

```diff
@@ -5,11 +5,11 @@
     "B", "C", "D", "F", "G",
     "H", "J", "K", "L", "M",
     "N", "P", "Q", "R", "S",
-    "T", "V", "W", "X", "Y", "Z", 
+    "T", "V", "W", "X", "Y", "Z",
 ]
 
 number_transforms = {
-    "O": "0",
+    # "O": "0",
```

Comment:
revert?

Reply:
That was intentional, see the last line of my summary - zero is not an allowed character (for CA at least).